### PR TITLE
Add sortType docs and demo

### PR DESCRIPTION
**Pull request by adlaird**
Resolves Issue #31 

-Added checkbox to sorting demo to toggle mulitple/single sorting options
-Added sortType to defaults.js and README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,6 +148,7 @@ The data table can be created with the following `options`:
 - `rowHeight`: Row height in pixels, necessary if using lazy rendering (default: `30`)
 - `scrollbarV`: Enable vertical scrollbars (default: `true`)
 - `selectable`: Whether users can select items (default: `false`)
+- `sortType`: Whether to allow sorting by multiple columns or only 1 (default: `multiple`)
```
Comment (jonshaffer) [inline README.md]: can you add that 'single' is the other option here?

File: src/defaults.js [hunk 2]
```diff
@@ -63,6 +63,9 @@ export const TableDefaults = {
   // if you can reorder columns
   reorderable: true,
 
+  // sorting by multiple columns or only one at a time
```
Comment (jonshaffer) [inline src/defaults.js]: can we use
// sorting by 'multiple' or 'single' columns
?

File: README.md (hunk 1 above)
Comment (adlaird) [inline README.md]: updated

File: src/defaults.js (hunk 2 above)
Comment (adlaird) [inline src/defaults.js]: updated

**Pull request merged**